lib/KCard/index.vue

```diff
@@ -418,4 +487,23 @@
     }
   }
 
+  .horizontal-with-large-thumbnail-and-right-alignment {
```

Comment:
Relatedly to https://github.com/learningequality/kolibri-design-system/pull/706/files#r1709907101, it seems to me this needs to be cleaned up. `horizontal-with-large-thumbnail` already contains styles related to left/right alignment in its `thumbnail-align-left` and `thumbnail-align-right`.